update-arm-images.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${CLIENT_ID}" ]] && echo "CLIENT_ID not set" >&2 && exit 1;
+[[ -z "${CLIENT_SECRET}" ]] && echo "CLIENT_SECRET not set" >&2 && exit 1;
+
+# Pull base images once to avoid pull limit in dockerhub
+docker pull ubuntu:18.04
+
+while IFS= read -r VERSION
+do
+if ! [[ "$VERSION" < "7.0.0" ]]; then
+  export VERSION=${VERSION}
+  ./build-arm-images.sh
```

Comment:
not the most robust way to access these scripts. I suggest you use the `D=` convention